integration-tests/tests/lock-context.test.ts

```diff
@@ -0,0 +1,93 @@
+import 'dotenv/config'
+import { describe, expect, test } from 'vitest'
+
+import {
+  type CastAs,
+  decrypt,
+  encrypt,
+  newClient,
+} from '@cipherstash/protect-ffi'
+
+const encryptConfig = {
+  v: 1,
+  tables: {
+    users: {
+      email: {
+        cast_as: 'text' as CastAs, // FIXME: do we need the as ?
```

Comment:
I think I'd prefer a [const assertion](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions) here.

Reply:
Can you explain what you mean?